.github/workflows/release-publish.yml

```diff
@@ -200,26 +203,20 @@ jobs:
     # `LAST_TAG` is evaluated using `git describe`, which is the name of the git tag before the release tag
     # `NUM_COMMITS` is the total number of commits "between" LAST_TAG and GIT_TAG
     #
-    # Release highlights are extracted from `site/docs/try/releases.md`, highlights for the current
-    # release is the text between the first two lines starting with `## ` in `releases.md`.
-    #
     # "Full Changelog" is the output of a `git log` considering the commits "between" LAST_TAG and
     # GIT_TAG, removing the commits by `dependabot-preview` and `nessie-release-workflow`.
     # Also removes commits that start with `[release] `.
     #
     # The final markdown is just a `cat` of the above information including some basic markdown formatting.
     #
-    - name: Prepare Nessie release in GitHub
-      id: prep_release
+    - name: Prepare Nessie release for GitHub
       env:
         GIT_TAG: nessie-${{ env.RELEASE_VERSION }}
```

Comment:
earlier we are now exporting `GIT_TAG` but here we are injecting it from scratch again